src/index.html

```diff
@@ -12,7 +12,54 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page__body">
+    <!-- <h1>Stars</h1> -->
+    <section class="stars stars--0">
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+    </section>
+
```

Comment:
There's an empty line here. Remember to add empty lines between multiline sibling blocks of HTML to add some "air" and simplify reading.